src/import-utils.ts

```diff
@@ -43,19 +41,21 @@ export async function inlineHtmlImport(
     analyzer: Analyzer,
     document: Document,
     linkTag: ASTNode,
-    stripImports: Set<UrlString>,
+    stripImports: Set<ResolvedUrl>,
     docBundle: AssignedBundle,
     manifest: BundleManifest,
     enableSourcemaps: boolean,
     rewriteUrlsInTemplates?: boolean,
     excludes?: string[]) {
   const isLazy = dom5.getAttribute(linkTag, 'rel')!.match(/lazy-import/i);
   const rawImportUrl = dom5.getAttribute(linkTag, 'href')!;
-  const importUrl = urlLib.resolve(document.url, rawImportUrl);
-  if (!analyzer.canResolveUrl(importUrl)) {
+  const importUrl = isAbsolutePath(rawImportUrl) ?
```

Comment:
Looks like this happens 3 times. Make a function?

Reply:
I delegate to `analyzer.urlResolver.resolve(document.url, rawImportUrl)` now which does this same job.